<launch>

  <arg name="joy_dev" default="/dev/input/js0"/>
  <arg name="joy_comms_config_file" default="$(find mcu_control)/config/joy_comms_test_config.yaml"/>

    <node name="joy_comms" pkg="mcu_control" type="joy_comms_control_node" output="screen">
      <rosparam command="load" file="$(arg joy_comms_config_file)"/>
    </node>

    <node name="joy_test" pkg="mcu_control" type="JoyTestNode.py" output="screen">
    </node>

</launch>